exegol/console/cli/actions/GenericParameters.py

```diff
@@ -196,7 +196,7 @@ def __init__(self, groupArgs: List[GroupArg]):
                             action="append",
                             default=[],
                             dest="ports",
-                            help="Share a network port between host and exegol (format: --port [<host_ipv4>:]<host_port>[:<container_port>][:<protocol>]. This configuration will disable the shared network with the host.",
+                            help="Share a network port between host and exegol (format: --port [<host_ipv4>:]<host_port>[-<end_host_port>][:<container_port>[-<end_container_port>]][:<protocol>]. This configuration will disable the shared network with the host.",
```

Comment:
would it be possible to defined a variable called `port_sharing_syntax` and have the help dialog become `Share a network [...] (format: {port_sharing_syntax}). This configuration[...]. Then, the variable could also be used later on when raising critical messages

Reply:
Ok, got it